device_doctor/tool/infra-dialog/infra-dialog.xcodeproj/project.pbxproj

```diff
@@ -249,6 +249,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = S8QB4VV633;
 				INFOPLIST_FILE = "infra-dialogUITests/Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
`Logger` API is on iOS 14 and higher, we are running this on iOS 16 so just bump the minimum.